data/rulei/rulei.txt

```diff
@@ -100,13 +122,21 @@ mission "Umbral Reach"
 		has "First Contact: Rulei: offered"
 		has "visited system: L-118"
 	on enter "Kanguwa"
+		event "antumbral reach label"
+		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Antumbral Reach", and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`
```

Comment:
```suggestion
		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Antumbral Reach," and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`
```

Reply:
For some reason this didn't get resolved?

Reply:
The original line was right, the comma is supposed to be outside of the quotation.